D.sublime-build

```diff
@@ -7,31 +7,32 @@
   [
     {
       "name": "Run",
-      "cmd": ["rdmd", "-g", "-debug", "$file"]
+      "working_dir": "$file_path",
+      "cmd": ["rdmd", "-g", "-debug", "$file_name"]
     },
     {
       "name": "dub",
-      "working_dir": "$project_path",
```

Comment:
If you create a sublime project file, you can have some settings specific for that project. Normally, the project file sits in the root of the project. So it sounds reasonable to use `$project_path` if found, and to use `$folder` otherwise, like in the other PR.